### PR TITLE
Add cyclic replication and configuration file in TiCDC deploy doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

+ Add cyclic replication
+ Add task configuration file

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3745 https://github.com/pingcap/docs-cn/pull/3666
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
Please help review this PR, thanks~ @ran-huang

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -351,3 +364,151 @@ For nodes other than owner nodes, executing the above command will return the fo
...
+
+![TiCDC cyclic replication](/media/cdc-cyclic-replication.png)
+
+To use the cyclic replication feature, you need to configure the following parameters for the replication task upon the task creation.
+
++ `--cyclic-replica-id`: Specifies the data source (to be written) ID of the upstream cluster. Each cluster ID must be unique.
++ `--cyclic-filter-replica-ids`: Specifies the data source ID to be filtered, which is usually the downstream cluster ID.
++ `--cyclic-sync-ddl`: Determines whether to replicate DDL statements to the downstream. DDL replication can only be enabled in the TiCDC component of one cluster.
+
+Take the following steps to create a cyclic replication task:
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:432]:
```suggestion
To create a cyclic replication task, take the following steps:
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -351,3 +364,151 @@ For nodes other than owner nodes, executing the above command will return the fo
...
+        --pd="http://${PD_C_HOST}:${PD_C_PORT}" \
+        --cyclic-replica-id 3 \
+        --cyclic-filter-replica-ids 1 \
+        --cyclic-sync-ddl false
+    ```
+
+### Usage notes
+
++ Before creating the cyclic replication task, you must execute `cdc cli changefeed cyclic create-marktables` to create the mark tables for the cyclic replication.
++ Tables with cyclic replication enabled only contain the [a-zA-Z0-9_] characters.
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:509]:
```suggestion
+ Tables with cyclic replication enabled only contain the `[a-zA-Z0-9_]` characters.
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -351,3 +364,151 @@ For nodes other than owner nodes, executing the above command will return the fo
...
++ Before creating the cyclic replication task, the tables for the task must be created.
++ After enabling the cyclic replication, you cannot create a table that will be replicated by the cyclic replication task.
++ To perform online DDL operations, ensure the following requirements are met:
+    - The TiCDC components of multiple clusters form a one-way DDL replication chain, which is not cyclic. For example, in the example above, only the TiCDC component of cluster C disables `sync-ddl`.
+    - DDL operations must be performed on the cluster that is the starting point of the one-way DDL replication chain.
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:514]:
```suggestion
    - DDL operations must be performed on the cluster that is the starting point of the one-way DDL replication chain, such as cluster A in the example above.
```

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -351,3 +364,151 @@ For nodes other than owner nodes, executing the above command will return the fo
...
+## Task configuration file
+
+This section introduces the configuration of a replication task.
+
+```toml
+# Specifies whether the database names and tables in the configuration file are case-sensitive.
+# The default value is true.
+# This configuration item affects configurations related to filter and sink.
+case-sensitive = true
+[filter]
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply ticdc/manage-ticdc.md:378]: Done.

**Comment by ran-huang** [comment]
@amyangfei @overvenus PTAL, thanks~

**Pull request merged**